dozer/cogs/toa.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, api_key, aiohttp_session, base_url="https://theorangealliance
         tries = 0
         while True:
             try:
+                print(type(self.http))
```

Comment:
Should probably remove this (hopefully) debug statment

Reply:
Yup